Makefile.in

```diff
@@ -67,7 +67,7 @@ endif
 
 build-multilib: build-multilib-m32 build-multilib-m64
 
-build-multilib-m32: config.status-multilib-m32
+build-multilib-m32: display-build-env config.status-multilib-m32
```

Comment:
Should we also remove `config.status-multilib-m32` from here and do an explicit `$(MAKE) config.status-multilib-m32` as the first line of this target?

Reply:
There's no need to create a recursive ${MAKE} in this case.  When we make 'config.status-multilib-m32', it won't print anything.  So, it won't interfere with the printing of 'display-build-env' even if the two are built in parallel.